gooddata-sdk/gooddata_sdk/catalog/user/entity_model/user.py

```diff
@@ -65,9 +87,21 @@ class CatalogUserAttributes(Base):
 class CatalogUserRelationships(Base):
     user_groups: Optional[CatalogUserGroupsData] = None
 
+    def add_user_groups(self, user_groups: list[CatalogUserGroup]) -> None:
+        if not self.user_groups:
+            self.user_groups = CatalogUserGroupsData(data=user_groups)
+        else:
+            self.user_groups.data.extend(user_groups)
+
+    def replace_user_groups(self, user_groups: list[CatalogUserGroup]) -> None:
+        if not self.user_groups:
+            self.user_groups = CatalogUserGroupsData(data=user_groups)
+        else:
+            self.user_groups.data = user_groups
+
     @property
     def get_user_groups(self) -> List[str]:
```

Comment:
Because it is a property, I would expect a different method name, without `get_` prefix.
If it has not been released yet (2.2 or before), I vote for renaming it.

Reply:
I am afraid that this was already released, but I think we can rename it and mention the change in release notes. Do you agree?

Reply:
Let's do it now, later, if it is used by many users, we never do it.